### PR TITLE
Milestone 2: Work and about section

### PR DESCRIPTION
**Pull request by cherelemma**
#Portfolio: Mobile First version
In this milestone of the portfolio: the mobile-first project task, the following features were added;
- Add recent works section and work history cards
- Add about us and skills sections
- Add style for all the above-mentioned sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
+          <hrx class="tags">
+            <ul>
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hxr>
```
Comment (Owusu-Desmond) [inline index.html:59]: - [ ] Please kindly check the style of this button, from the project requirements, you should stick to the design as much as possible. But as you can see you have some differences in the design. You can see the design below and compare it with your project design. :pray:

| Your Design (there is a line break in between the `see` and `project` |Figma Design (No line break in between the `see` and `project`) |
| :---:   | :---: | 
| Screenshot from 2022-10-28 21-10-24 | Screenshot from 2022-10-28 21-14-53
 |

File: index.html [hunk 2]
```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
+          
+          <hrx class="tags">
+            <ul>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hrx>
```
Comment (Owusu-Desmond) [inline index.html:77]: - [ ] Please make sure your `see project` button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 21-18-21

For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

File: index.html [hunk 3]
```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html:93]: - [ ] Please make sure your `see project` button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 21-18-21

For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

File: index.html [hunk 4]
```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html:108]: - [ ] Please make sure your `see project` button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 21-18-21

For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

File: index.html [hunk 5]
```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html:123]: - [ ] Please make sure your `see project` button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 21-18-21

For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

File: index.html [hunk 6]
```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html:138]: - [ ] Please make sure your `see project` button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 21-18-21

For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

File: index.html [hunk 7]
```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html:153]: - [ ] Please make sure your `see project` button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 21-18-21

For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

File: index.html [hunk 8]
```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html:168]: - [ ] Please make sure your `see project` button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 21-18-21

For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @cherelemma , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bienvenuushindi — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @cherelemma   👋, 

Well done implementing the requested changes 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
:+1: No linter errors✔️
:+1: Good UI design✔️
:+1: Good PR title

### Optional Suggestions
N/A
**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**